src/parsers/json-parser.ts

```diff
@@ -6,7 +6,7 @@ export class JsonParser implements TranslationFileParser {
     parseFrom(fileContent: string): Promise<JsonFile> {
         const buildMap = (obj: any, parentPath?: string) => {
             for (const [key, value] of Object.entries(obj)) {
-                const path = parentPath ? `${parentPath}.${key}` : key;
+                const path = parentPath ? `${parentPath}[--]${key}` : key;
```

Comment:
This is a temporary delimiter, which is only used within the context of the action. In other words, this would never persist when being written back out - right?